guides/common/modules/snip_steps-needs-reboot.adoc

```diff
@@ -18,7 +18,7 @@ ifndef::foreman-deb[]
 endif::[]
 endif::[]
 ifdef::satellite[]
-. If the previous command told you to reboot, then reboot the system:
+. If the {foreman-maintain} command told you to reboot, then reboot the system:
```

Comment:
Perhaps be even more explicit and say
```suggestion
. If the previous {foreman-maintain} command told you to reboot, then reboot the system:
```

Reply:
This snippet includes the older way of determining reboot and does not contain the foreman-maintain command anywhere in it. Is this the wrong spot to add this?

Reply:
> This snippet includes the older way of determining reboot and does not contain the foreman-maintain command anywhere in it.

That is guarded behind `ifdef` statements for upgrade procedures without foreman-maintain.

Note that if you read the rendered documentation then it's always included in a procedure where step n is `satellite-maintain` and step n+1 this statement. I can't verify the original report.

Reply:
You could also write "If {foreman-maintain} in the previous step told you…"